src/utils/config_utils/tests/config_utils_ut.cpp

```diff
@@ -30,7 +30,10 @@ static const char* validConfigContentStr =
         R"("aduShellTrustedUsers": ["adu","do"],)"
         R"("manufacturer": "device_info_manufacturer",)"
         R"("model": "device_info_model",)"
-        R"("compatPropertyNames": "manufacturer,model",)"
+        R"("compatProperties": { )"
+        R"(   "deviceManufacturer": "contoso", )"
+        R"(   "deviceModel": "camera", )"
```

Comment:
Do we still need "manufacturer" and "model" property below?

Reply:
no, not needed. removing